File: Capitulo_3/Python/A_Exemplos_pag_54_ate_69/8_loja_senhor_Tanaka.py
numeroCamisas = int(input("Quantidade de camisas: "))
valorCamisa = 12.50
valorFinal = numeroCamisas * valorCamisa

if numeroCamisas <= 5:
    valorFinal = valorFinal * (1 - 3/100)
else:
    if numeroCamisas <= 10:
        valorFinal = valorFinal * (1 - 5/100)
    else:
        valorFinal = valorFinal * (1 - 7/100)

print(f"Valor final: R$ {valorFinal:.2f}")

# pag 68

File: Capitulo_3/JavaScript/A-Exemplo-pag-54-ate-69/8-loja-senhor-Tanaka.js
const prompt = require('prompt-sync')()

const numeroCamisas = Number(prompt('Quantidade de camisas: '))
const valorCamisa = 12.50
let valorFinal = numeroCamisas * valorCamisa

if (numeroCamisas <= 5){
    valorFinal = valorFinal * (1 - (3/100))
}else{
    if (numeroCamisas <= 10){
        valorFinal = valorFinal * (1 - (5/100))
    }else{
        valorFinal = valorFinal * (1 - (7/100))
    }
}

console.log(`Valor final: R$ ${valorFinal.toFixed(2)}`)
    